app/models/miq_ae_class_copy.rb

```diff
@@ -79,13 +79,17 @@ def add_fields
 
   def validate
     dest_class = MiqAeClass.find_by_fqname("#{@target_ns_fqname}/#{@target_name}")
+    $miq_ae_logger.info("Destination class: #{dest_class}")
```

Comment:
@mkanoor Does the logging belong in the automation log or evm log or somewhere else?

Reply:
@d-m-u Discussed with @mkanoor and agreed that this logging belongs in the evm.log so these calls should be using `$log`.